meta-resin-common/recipes-containers/docker/docker_git.bb

```diff
@@ -18,27 +18,18 @@ DESCRIPTION = "Linux container runtime \
  subtle and/or glaring issues. \
  "
 
-SRCREV = "20f81dde9bd97c86b2d0e33bbbf1388018611929"
-SRCBRANCH = "v1.10.3"
+SRCREV = "d2be68c9bf542a57904702d06ceaa0d2c462745c"
+SRCBRANCH = "docker-binaries-consolidation-incremental"
```

Comment:
we have to finalise the branch/tag in that repo before merging. This was a WIP branch from yossi